src/api/models/article.js

```diff
@@ -1,59 +1,58 @@
-const Article = (sequelize, DataTypes) => {
-  const article = sequelize.define('Articles',
+import slug from 'slug';
+
+const THIRTY_SIX = 36;
+const SIX = 6;
+const ZERO = 0;
+
+export default (sequelize, DataTypes) => {
+  const Article = sequelize.define('Articles',
     {
       id: {
         allowNull: false,
-        unique: true,
+        autoIncrement: true,
         primaryKey: true,
-        type: DataTypes.UUID,
-        defaultValue: DataTypes.UUIDV4
-      },
-      title: {
-        type: DataTypes.STRING,
-        allowNull: false
-      },
-      description: {
-        type: DataTypes.STRING,
-        allowNull: false
-      },
-      slug: {
-        type: DataTypes.STRING,
+        type: DataTypes.INTEGER
+      },
+      title: { type: DataTypes.STRING, allowNull: false },
+      description: { type: DataTypes.TEXT, allowNull: false },
+      body: { type: DataTypes.TEXT, allowNull: false },
+      slug: { type: DataTypes.STRING, allowNull: false },
+      coverImage: { type: DataTypes.TEXT, allowNull: false },
+      author: {
         allowNull: false,
-        unique: true
+        type: DataTypes.INTEGER,
+        references: { model: 'Users', key: 'id' }
       },
       category: {
-        type: DataTypes.STRING,
         allowNull: false,
-      },
-      body: {
-        type: DataTypes.TEXT,
-        allowNull: false
-      },
-      coverImage: {
-        type: DataTypes.TEXT,
-        allowNull: true
-      },
-      tagList: {
-        type: DataTypes.ARRAY(DataTypes.TEXT),
-        allowNull: false
-      },
-      userId: {
         type: DataTypes.INTEGER,
-        references: {
-          model: 'Users',
-          key: 'id'
-        },
-        onDelete: 'CASCADE',
-        onUpdate: 'CASCADE',
```

Comment:
@murediane why have you removed these:

onDelete: 'CASCADE',	
onUpdate: 'CASCADE',

Reply:
They are not removed  Frank did not have a relationship with categories model so I had to move them to the appropriate place. but they are still in the file